scripts/modal.js

```diff
@@ -0,0 +1,223 @@
+const project1 = {
```

Comment:
Please align the project title and close button vertically on the mobile view. Kindly see the screenshot of your project below.


![popup-mobile](https://user-images.githubusercontent.com/35504149/189412191-9dc9ae66-0e87-4721-a348-79a6589b48aa.png)

Reply:
Hi **@SajjadAhmad14**  👋,

Thank you so much for the highlights  👏 however, am having a hard time comprehending your comment on the alignment of the project title with the close button.

I actually followed the design as much as I can, so kindly asking can you please explain it more? 

### Figma Design 
![image](https://user-images.githubusercontent.com/68030297/189414959-4684c5a1-7c48-48b7-b5ef-a14e28f3cf87.png)

### My Design
![image](https://user-images.githubusercontent.com/68030297/189415088-bb04af6d-a682-49bd-9cd2-8214eaee665a.png)

and once again **thank you very much**.